src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerProvidersLabel.xaml.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// Interaction logic for PackageManagerProvidersLabel.xaml. Its DataContext is
+    /// PackageItemListViewModel.
+    /// </summary>
+    public partial class PackageManagerProvidersLabel : UserControl
+    {
+        public PackageManagerProvidersLabel()
+        {
+            InitializeComponent();
+
+            this.DataContextChanged += PackageManagerProvidersLabel_DataContextChanged;
+        }
+
+        private void PackageManagerProvidersLabel_DataContextChanged(object sender, DependencyPropertyChangedEventArgs e)
+        {
+            _textBlock.Inlines.Clear();
```

Comment:
Note that we have to generate the UI in the code instead of in xaml since 
- we need to generate multiple runs, since some are texts, while others are links;
- and TextBox.Inlines does not support databinding;

Also, we have to handle the format string ourselves to replace "{0}" with the list of links.